### PR TITLE
Timing

### PR DESCRIPTION
**Pull request by Asa-Kosto-QTM**
# Description

Updated timing data and added a function that reliably brings PHIR estimated transport time to within 10-15% of the true transport time.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [X] I have performed a self-review of my code
- [ ] I have added tests that prove my fix is effective or that my feature works
	- N/A, all tests require internal resources for verification.

### CODE REVIEW

**Review by qartik — CHANGES_REQUESTED** [review]

File: pytket/phir/machine.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ class MachineTimings:
     tq_time: float
     sq_time: float
     qb_swap_time: float
+    measurement_prep_transport_time: float
```
[the remark this answers is not in the document]
Comment (Asa-Kosto-QTM) [reply pytket/phir/machine.py:28]: i think there should be some indication that it is transport specifically to prepare for measurement, not the measurement itself. maybe meas_prep_time?

[the remark this answers is not in the document]
Comment (qartik) [reply pytket/phir/machine.py:28]: That works too. I wasn't sure of the importance of all the keywords in there.

File: pytket/phir/phirgen_parallel.py [hunk 2]
```diff
@@ -280,6 +280,33 @@ def format_and_add_primary_commands(
...
+            match first_op:
+                case "RZ" | "R1XY":
+                    adjustment += machine.sq_time
+                case "RZZ":
+                    adjustment += machine.tq_time
+        if "mop" in op and op["mop"] == "Transport":
+            duration = op["duration"][0] + adjustment
+            units = op["duration"][1]
+            transport_info = (duration, units)
+            op["duration"] = transport_info
```
Comment (qartik) [inline pytket/phir/phirgen_parallel.py:306]: How about this?
```suggestion
            duration, units = op["duration"]
            op["duration"] = duration + adjustment, units
```

**Review by qartik — DISMISSED** [review]
Looking good, leaving a minor suggestion.

**Review by qartik — DISMISSED** [review]

**Review by qciaran — APPROVED** [review]
Alt Text
LGTM
Code seems minimal and sensical. Output looks as expected on my end.

**Review by neal-erickson — CHANGES_REQUESTED** [review]
Waiting on feedback from Alex

**Review by qartik — APPROVED** [review]

**Pull request closed without merging**